packages/@react-aria/calendar/src/useCalendarTableHeader.ts

```diff
@@ -1,11 +0,0 @@
-interface CalendarTableHeaderAria {
-  columnHeaderProps: {scope?: 'col'}
-}
-
-export function useCalendarTableHeader(): CalendarTableHeaderAria {
-  return {
-    columnHeaderProps: {
-      scope: 'col'
```

Comment:
I think this is unnecessary? <th> elements in the first row of a table should apply as column headers automatically I believe. https://www.w3.org/TR/2011/WD-html5-author-20110809/the-th-element.html#attr-th-scope